### DISCUSSION TITLE
Where is the "Custom sort list" saved?

### DISCUSSION THREAD

**Discussion by MustafaEsatTemel** [SebastianMC/obsidian-custom-sort#146] [Q&A]
Firstly sorry for my english. I want to reset my pc, but I don't want to lose my sort list. Is there any data file for sort list ?

**Comment by SebastianMC** [comment]
Hi @MustafaEsatTemel 

technically speaking, the sorting information can be stored in the below locations:

1. As an explicit sorting specification in YAML frontmatter of note(s) (aka in properties of notes). Because this is part of your notes, by definition, this sorting information can't be lost until you explicitly remove them by editing or removing the relevant notes. At the same time, this is probably not the case for which you are asking -if you wrote the sorting spec in your notes you would know this.
2. If you use the manual drag & drop sorting via Obsidian Bookmarks integration, then the sorting list is part of Obsidian bookmarks, stored by the Bookmarks subsystem as part of your vault. This file is stored as part of your Obsidian vault in a file in a hidden folder `/.obsidian/bookmarks.json`. In result, if you retain / move / copy the Obsidian vault as-is, including the hidden folder(s), the sorting information is retained.

In other words, sorting information is stored as part of the Obsidian vault, this or either way. If you keep the vault intact (including hidden folders), the sorting information is kept / moved / copied / restored from backup correctly.
